packages/contracts-bedrock/scripts/deploy/VerifyOPCM.s.sol

```diff
@@ -657,4 +759,45 @@ contract VerifyOPCM is Script {
         // Return computed path, relative to the contracts-bedrock directory.
         return string.concat("forge-artifacts/", sourceName, ".sol/", _contractName, ".json");
     }
+
+    /// @notice Validates that all getter functions in the OPContractsManager ABI are accounted for
+    ///         in the expectedGetters mapping. This ensures we don't miss any new getters that
+    ///         might be added to the contract.
+    function _validateAllGettersAccounted() internal {
+        // Get all function names from the OPContractsManager ABI
+        string[] memory allFunctions = abi.decode(
+            vm.parseJson(
+                Process.bash(
+                    string.concat(
+                        "jq -r '[.abi[] | select(.type == \"function\" and .stateMutability == \"view\" and (.inputs | length) == 0) | .name]' ",
+                        _buildArtifactPath("OPContractsManager")
+                    )
+                )
+            ),
+            (string[])
+        );
```

Comment:
We should put this into a utility function, called `_getOpcmGetters()`, so that we can reuse it above.

Reply:
Solved in 2de2f02d47fb52247e66655e338e64413c42adfe by creating a helper function